pcl_ros/include/sixriver/gpu/filters/voxel_grid.h

```diff
@@ -595,10 +664,10 @@ namespace sixriver
       inline void 
       setMinimumPointsNumberPerVoxel (unsigned int min_points_per_voxel) { min_points_per_voxel_ = min_points_per_voxel; }
 
-	  /** \brief Return the minimum number of points required for a voxel to be used.
+	    /** \brief Return the minimum number of points required for a voxel to be used.
```

Comment:
Is this an indentation fix?

Reply:
Yes, I found the indention was wrong (not consistent with other functions)